hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Reconciler.java

```diff
@@ -117,4 +124,42 @@ private void explicitlyReconcileTasks(SchedulerDriver driver) {
 
     driver.reconcileTasks(tasks);
   }
+
+  private class ReconcileThread extends Thread {
+    private static final int MILLIS_FROM_SECONDS = 1000;
```

Comment:
Not trying to imply anything about this being your first day developing software :) Your code is awesome. It's just that we tend to be extremely picky and as careful as possible with the project to avoid any issues. If it's used in one place fine, but let's avoid duplicating commonly used constants. I also prefer to break these out constants into separate files as needed. We might consider putting this in HdfsConstants if we will have to conversions in other places in future.